drafts/generating-css-from-scratch-with-postcss.md

```diff
@@ -0,0 +1,113 @@
+# Generating CSS from scratch with PostCSS
+
+For a soon-to-be open-sourced project I had to generate a CSS stylesheet from some data.
```

Comment:
Maybe open even stronger? E.g. "For an open source project I needed to generate CSS based on user data, and soon realised old fashioned string templating wasn't going to cut it." or something?

Reply:
Hm I'll think about it :-)